docs/mindsdb-docs/docs/sql/api/select_files.md

```diff
@@ -1,14 +1,15 @@
-# `#!sql SELECT files.[file]` Statement
+# `#!sql SELECT from files.[file_name]` Statement
 
 ## Description
 
-The `#!sql SELECT from files.[file]` statement is used to select a `#!sql [file]` as a datasource. The main use is to create a predictor from a file that has been uploaded to MindsDB via the [MindsDB Editor](/connect/mindsdb_editor/).
+The `#!sql SELECT from files.[file_name]` statement is used to select data from a file.
```

Comment:
We talk about sql statement, but statement itself is not sql-valid. That is confusing me. Let's add `*` to statement:
```
SELECT * FROM files.[file_name]
```

Reply:
Or may be split it to
```
`#!sql SELECT` from `#!sql files.[file_name]`
```

Reply:
Changed it to `SELECT * FROM files.[file_name]`.